tests/src/main/java/ca/bc/gov/open/jagefilingapi/qa/restdata/Payload.java

```diff
@@ -0,0 +1,33 @@
+package ca.bc.gov.open.jagefilingapi.qa.restdata;
+
+public class Payload {
+
+    public static String submitDocumentProperties() {
+        return "{\n" +
+                "  \"documentProperties\": {\n" +
+                "    \"type\": \"string\",\n" +
+                "    \"subType\": \"string\",\n" +
+                "    \"submissionAccess\": {\n" +
+                "      \"url\": \"string\",\n" +
+                "      \"verb\": \"GET\",\n" +
+                "      \"headers\": {\n" +
+                "        \"additionalProp1\": \"string\",\n" +
+                "        \"additionalProp2\": \"string\",\n" +
+                "        \"additionalProp3\": \"string\"\n" +
+                "      }\n" +
+                "    }\n" +
+                "  },\n" +
+                "  \"navigation\": {\n" +
+                "    \"success\": {\n" +
+                "      \"url\": \"string\"\n" +
+                "    },\n" +
+                "    \"error\": {\n" +
+                "      \"url\": \"string\"\n" +
+                "    },\n" +
+                "    \"cancel\": {\n" +
+                "      \"url\": \"string\"\n" +
+                "    }\n" +
+                "  }\n" +
```

Comment:
code climate is also complaining about size of this function... so maybe we could separate out this navigation section into its own string

```
String navigationSection = "blah";
```

and then use it within this so that would drop the complexity of this function and allow you to refactor

Reply:
Not applicable. Refactored. Thanks